AccessHandle.md

```diff
@@ -184,12 +184,6 @@ observe changes done through the new API, even if a lock is still being held.
 
 ## Open Questions
 
-### Naming
-
-The exact name of the new methods hasn’t been defined. The current placeholder
-for data access is *createAccessHandle()* and *createSyncAccessHandle()*.
-*createUnflushedStreams()* and *createDuplexStream()* have been suggested.
```

Comment:
I'm surprised that the names for the streams methods have been decided, can't this wait until they have been implemented?

Reply:
Scratch that, I misunderstood the naming section.